Common/Domain.UnitTests/Domain.UnitTests.fsproj

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Please retain this project even if you remove the code following our discussion. There are some [domain tests that reside in engine.core](https://github.com/parthopdas/tddstud10/tree/master/Engine.Core.UnitTests/Domain) and in the future we will move them into this assembly.